e2eshark/run.py

```diff
@@ -463,15 +463,20 @@ def runCodeGeneration(
 #    bf16: atol=1e-05, rtol=1.6e-02
 #    fp16: atol=1e-05, rtol=1e-03
 #    fp32: atol=1e-05, rtol=1.3e-06
-def getTolerances(torchdtype):
-    if torchdtype == torch.bfloat16:
+def getTolerances(args, torchdtype):
```

Comment:
@afalkenberg1 thank you for adding this option. I think we should make it override default regardless of the dtype. This is a usewr option to override the default. So, if rtol and atol has been passed, then use it whatever it is.

Reply:
ok I am dropping the case